src/components/WebexMeetingParticipant/WebexMeetingParticipant.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {DestinationType} from '@webex/component-adapter-interfaces';
+import Spinner from '../generic/Spinner/Spinner';
+
+import Icon from '../generic/Icon/Icon';
+import {
+  useMembers,
+  useMe,
+  useOrganization,
+  usePerson,
+} from '../hooks';
+import WebexAvatar from '../WebexAvatar/WebexAvatar';
+import webexComponentClasses from '../helpers';
+
+/**
+ * Displays a webex meeting participant.
+ *
+ * @param {object} props  Data passed to the component
+ * @param {string} props.className  Custom CSS class to apply
+ * @param {string} props.meetingID  ID of the meeting for which to get members
+ * @param {boolean} props.displayStatus  Whether or not to display the user's status
+ * @param {string} props.memberID  ID of the member for which to display avatar
+ * @param {object} props.style  Custom style to apply
+ * @returns {object} JSX of the component
+ *
+ */
+export default function WebexMeetingParticipant({
+  className,
+  meetingID,
+  displayStatus,
+  memberID,
```

Comment:
Let's call this `personID` because the expectation is that this is the Webex person ID. There is a different member ID, that is per person per meeting that is random and this component doesn't use that